libs/books/feature/src/lib/reading-list/reading-list.component.ts

```diff
@@ -10,9 +12,20 @@ import { getReadingList, removeFromReadingList } from '@tmo/books/data-access';
 export class ReadingListComponent {
   readingList$ = this.store.select(getReadingList);
 
-  constructor(private readonly store: Store) {}
+  constructor(private readonly store: Store, private readonly snackBar: MatSnackBar) {}
 
   removeFromReadingList(item) {
     this.store.dispatch(removeFromReadingList({ item }));
+    this.snackBar.openFromComponent(
```

Comment:
It looks like this would open simultaneously with the action that it needs to undo. Could these ever get out of sync?

Shouldn't this show when the action has been done?

Reply:
Oh, about that, no, It cant overlap or open simultaneously, as the snack bar is bound to the click on the buttons and not action or reducer, so we can dispatch the action without worrying about  the snack bar triggering again.